cardano-constitution/cardano-constitution.cabal

```diff
@@ -127,13 +127,3 @@ test-suite cardano-constitution-test
     , tasty-hunit
     , tasty-json
     , tasty-quickcheck
-
-executable create-json-envelope
-  import:         lang, ghc-version-support, os-support
-  hs-source-dirs: create-json-envelope
-  main-is:        Main.hs
-  build-depends:
-    , base
-    , cardano-api           ^>=8.48
-    , cardano-constitution
-    , plutus-ledger-api
```

Comment:
Is it ok for you to resurrect this and its accompanying Main.hs file with a `buildable:False` temporary solution?

Reply:
Trying if this works...